src/CodeControl.jl

```diff
@@ -0,0 +1,156 @@
+export @with_stdout, @cond, @cond_with_stdout, @bind_cond, stdout_style, stdout_styles
+
+"""
+     Dict containing predefined style sheets for `div.stdout`
+"""
+_predefined_css=Dict(
+
+    :default =>"""<style>
+         div.stdout {}
+         div.stdout pre {
+                color: #000;
+                    border-radius: 1px;
+                background-color: #dfd;
+                    border: 0px solid #ddd;
+                font-size: 65%;
+          }
+           </style>""",
+
+    :nowrap =>"""<style>
+         div.stdout {}
+         div.stdout pre {
+                color: #000;
+                border-radius: 1px;
+                background-color: #dfd;
+                border: 0px solid #ddd;
+                font-size: 65%;
+                word-break: normal !important;
+                word-wrap: normal !important;
+                white-space: pre !important;
+      	        overflow: auto;
+          }
+           </style>""",
+
+    :vintage =>"""<style>
+              div.stdout {}
+              div.stdout pre {
+                color: #2f2;
+                border-radius: 10px;
+                background-color: #333;
+                border: 5px solid #aaa;
+                font-size: 65%;
+          }
+          </style>""")
```

Comment:
Let's pick a favorite style and make that the only option.

Terminals always wrap, so I think that this should be the default.

Reply:
I am quite sure people will ask about `nowrap`  some time  :-}, but I don't insist. 

I think `:vintage` is too heavy on the eyes (btw the first CRT screens were indeed green) and doesn't behave well with printing. So I would opt for something light as in `:default`. 

In light of https://github.com/JuliaIO/Suppressor.jl/issues/35 we may expect that stderr will be available as well, so indeed I would opt for green-ish for "ok" output and red-ish  for error output.

But: do you foresee some preconfiguration for pluto styles ? Modified css for this then could go there.

Reply:
See https://github.com/JuliaIO/Suppressor.jl/pull/36